spec/machine/ARM/Arch_Kernel_Config_Lemmas.thy

```diff
@@ -119,5 +119,15 @@ lemma maxIRQ_1_plus_eq_Suc_machine[simp]:
   "unat (1 + maxIRQ :: machine_word) = Suc Kernel_Config.maxIRQ"
   by (simp add: Kernel_Config.maxIRQ_def)
 
+
+(* cacheLineBits conditions *)
+
+(* Folding cacheLineBits_val in C functions only works reliably if cacheLineBits is not 1 and
+   not too large to conflict with other values used inside cache ops.
+   10 is ptBits, which is only available after ExecSpec. Anything smaller than ptBits works. *)
+lemma cacheLineBits_sanity:
+  "cacheLineBits \<in> {2..10}"
```

Comment:
would it make sense to say ptBits for these instead of the `..10` or whatever and have a `[simplified ptBits_def]` or whatever on the lemma statement?

Reply:
oh right, "which is only available after ExecSpec". Hmm. Nitpick: "anything smaller than" and "not 1" don't gel together well

Reply:
Good point, have adjusted it slightly.